src/component/panels/InformationPanel.jsx

```diff
@@ -7,25 +7,95 @@ import {
   TableRow,
   TableCell,
 } from '../elements/Table';
-import { fromJcamp } from '../../data/data1d/Data1DManager';
+// import { ChartContext } from '../context/ChartContext';
+// import { fromJcamp } from '../../data/data1d/Data1DManager';
+
+const styles = {
+  input: {
+    backgroundColor: 'transparent',
+    border: 'none',
+    width: '',
+    height: '14px',
+  },
+};
+
+const filter = (input) => {
+  let escaped = input.replace(/[.*+?^${}()|[\\]\\\\]/g, '\\\\$&');
+  let regexp = new RegExp(escaped, 'i');
+  let tableContent = document
+    .getElementById('FilterAndInformationContainer')
+    .getElementsByClassName('InformationContainer')
+    ['0'].getElementsByClassName('Information');
+
+  for (let content of tableContent) {
+    // check the property string only
+    let labelContent = content.children[0].innerHTML;
+    if (labelContent.match(regexp)) {
+      content.style.display = 'flex';
+    } else {
+      content.style.display = 'none';
+    }
+  }
+};
```

Comment:
@michaelwenk
can you explain to my your code.
and also you should use Callback instead of regular function to prevent from initiate function object every time you call it.

you can call it what ever you want, and inside the function use setSearchSequence.


also at some case we have more than one spectrum and your code only handle one spectrum info data, so if we have more than one how you will handle it?

Reply:
Indeed it is a problem when we have many spectra and I think spectrum info should be desactivate with a message if there is more than one spectrum selected. At least for now

Reply:
-You not have to access to dom elements directly andthnig you need to update it to dom use state for that,
- you don't need to use regular expression here there are a short way for this using includes function in your case and it will give you the same result.

can you pealse check my last commit at you code and if you have question you can ask me when ever you want.